charts/config.yaml

```diff
@@ -0,0 +1,19 @@
+remote: origin
+since: HEAD~1
+github-instance: https://github.com
+lint-conf: charts/lintconf.yaml
+chart-yaml-schema: charts/chart_schema.yaml
+validate-chart-schema: true
+validate-yaml: true
+check-version-increment: false
+charts:
+  - charts/aws-ebs-csi-driver
+helm-extra-args: "--timeout 1000s"
+upgrade: true
+skip-missing-values: true
+namespace: kube-system
+release-label: release
+kubectl-timeout: 1000s
+skip-clean-up: false
+print-logs: true
+debug: true
```

Comment:
Is it not possible to put this file in the test directory? Can it be named something other than `config.yaml`?

Reply:
Also, `debug: true`?

Reply:
Done.